### ISSUE TITLE
Remove deprecated strings

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#1967]
leftovers from old speak implementation. Not sure if they migth be referred still from code, as speak implementation is partially still being used (from reading plans etc).

one_chapter
some_chapters
one_verse
some_verses
one_page
some_pages

**Comment by agrogers** [comment]
I wouldn't remove them unless they havent been translated. They could be
useful in the future.

___________________________
*www.TenOutOfTen.org* <https://www.TenOutOfTen.org>
***@***.***
(+61) 0477 174 193 (Australia)
(+95) 09 250018669 (Myanmar)

**Comment by tuomas2** [comment]
You are right, these strings are quite general and possible to be used.
BUT, even if I removed them, they would be stored in Transifex's "translation memory". So it does not really matter. If we keep them, they will add extra burden to new translators.

**Comment by agrogers** [comment]
Good point. If we can get them back easily then no harm in removing them.

___________________________
*www.TenOutOfTen.org* <https://www.TenOutOfTen.org>
***@***.***
(+61) 0477 174 193 (Australia)
(+95) 09 250018669 (Myanmar)
